infrastructure/k8s-config/clusters/kit-infrastructure/provisioner.yaml

```diff
@@ -9,6 +9,11 @@ spec:
     aws-cdk:subnet-type: Private
   securityGroupSelector:
     kit.sh/stack: KITInfrastructure
+  blockDeviceMappings:
+  - deviceName: /dev/xvda
+    ebs:
+      volumeSize: 100Gi
```

Comment:
do we really need this much memory ? Thinking in terms of Frugality principle for customers using this Infra.

What's causing disks to fill up ? If it's logs, we should probably have the CW logging enabled so we rotate the logs on the instances so disks won't fill up.

Reply:
100Gi is still pretty small. This is a mixture of container images that are pulled and logs for tests. We'll definitely rotate logs soon so that we don't breach this limit, but I think 100Gi is still pretty frugal but much more reasonable for the type of work the nodes are facilitating.

Reply:
ok, we can go ahead with this change for now but we should be addressing the root of the problem before we increase it further in future.
 My concern was that 100Gi won't be enough if root of the problem is not addressed. If it's log rotation, that should be addressed is what I was trying to convey. I had a backlog ticket cut for this in the past - https://github.com/awslabs/kubernetes-iteration-toolkit/issues/279 which should help flushing logs from nodes to CW.